src/scripts/OutSystems/OSUI/Patterns/VideoAPI.ts

```diff
@@ -128,4 +128,65 @@ namespace OutSystems.OSUI.Patterns.VideoAPI {
 
 		return result;
 	}
+
+	/**
+	 * Function that returns the state of a given video
+	 *
+	 * @export
+	 * @param {string} videoId
+	 * @return {*}  {string}
+	 */
+	export function VideoGetState(videoId: string): string {
```

Comment:
If was me, I renamed this method into **GetState** only, that way API namespace will be:
**_OutSystems.OSUI.Patterns.VideoAPI.GetState_** instead of **_OutSystems.OSUI.Patterns.VideoAPI. VideoGetState**...

Reply:
I always keep the same name as the SS action for a direct matching. However, in point of view on of calling the API, that makes more sense. I will change the name of methods.

Reply:
Done